pkgs/applications/misc/plover/default.nix

```diff
@@ -1,32 +1,98 @@
-{ lib, fetchFromGitHub, python3Packages, wmctrl, qtbase, mkDerivationWith }:
+{ lib, fetchFromGitHub, python310Packages, wmctrl, qtbase, mkDerivationWith }:
 
 {
   stable = throw "plover.stable was removed because it used Python 2. Use plover.dev instead."; # added 2022-06-05
 
-  dev = with python3Packages; mkDerivationWith buildPythonPackage rec {
+  dev = with python310Packages; let
+    plover_stroke = mkDerivationWith buildPythonPackage rec {
+      pname = "plover_stroke";
+      version = "1.1.0";
+
+      pyproject = true;
+
+      src = fetchFromGitHub {
+        owner = "openstenoproject";
+        repo = "plover_stroke";
+        rev = "${version}";
+        sha256 = "sha256-A75OMzmEn0VmDAvmQCp6/7uptxzwWJTwsih3kWlYioA=";
+      };
+
+      meta = with lib; {
```

Comment:
same here as below

Reply:
The attribute is at the end of the derivation